rpcs3/evdev_joystick_handler.cpp

```diff
@@ -168,6 +168,30 @@ void evdev_joystick_handler::Close()
     }
 }
 
+inline int Deadzone(f32 input)
+{
+	int	deadzone=g_evdev_joystick_config.deadzone;
+	float deadzone_a=128.0/(128.0-float(deadzone));
+	float deadzone_b=255.0*(1.0-deadzone_a);
+	if (input>=128-deadzone&&input<=128+deadzone)
```

Comment:
this should be
```cpp
if (input >= 128 - deadzone && input <= 128 + deadzone)
```
to match with the rest of the file's style, I'd like to see the same done with most of the rest of the changes in this PR.
(same with assignment, I'd like to see `int x = y;` instead of `int x=y;`)